llpc/context/llpcCompiler.cpp

```diff
@@ -1251,32 +1251,32 @@ unsigned GraphicsShaderCacheChecker::check(const Module *module, const unsigned
   Compiler::buildShaderCacheHash(m_context, stageMask, stageHashes, &fragmentHash, &nonFragmentHash);
   unsigned stagesLeftToCompile = stageMask;
 
-  if (stageMask & shaderStageToMask(ShaderStageFragment)) {
+  if (stageMask & (1 << lgc::ShaderStageFragment)) {
```

Comment:
Could we have this as a new function, e.g., `lgcStageToMask()`?

Reply:
Sure. Will modify.